site2/website/blog/2019-02-20-Apache-Pulsar-2-3-0.md

```diff
@@ -0,0 +1,145 @@
+---
+author: Matteo Merli
+authorURL: https://twitter.com/merlimat
+title: Apache Pulsar 2.3.0
+---
+
+The Apache Pulsar PMC is happy to announce the release of Pulsar 2.3.0. This
+is the result of huge effort from the community, with over 480 commits and
+a long list of new features, general improvements and bug fixes.
+
+These improvements have been across the board in all of Pulsar components,
+from new messaging features, to improved usability for Pulsar Functions
+and Pulsar IO.
+
+Check out the official <b>[release notes](/release-notes/#2.3.0)</b> for a
+detailed list of the changes, with link to the relevant pull-requests,
+discussions and documentation.
+
+Regarding new features introduced, I just want to highlight here a tiny
+subset between them:
+
+<!--truncate-->
+
+### Pulsar functions in Kubernetes
+
+It's now possible to use Kubernetes as the scheduler for Pulsar Functions.
+
+When a Pulsar cluster is configured to use Kubernetes, submitting a
+function, with the using CLI tools or REST API, will cause a
```

Comment:
typo. "with the CLI tools or REST API"

will cause a what?

Reply:
ups.